RecoEgamma/EgammaElectronProducers/python/electronPFIsolationDeposits_cff.py

```diff
@@ -26,10 +26,11 @@
 gedElPFIsoDepositGamma=isoDepositReplace('gedGsfElectronsTmp','pfAllPhotons')
```

Comment:
earlier in this file I see ``` electronPFIsolationDepositsSequence = cms.Sequence(``` which is not converted to a task.
I'd expect that once a file is touched to make a Sequence -> Sequence(Task) conversion, all sequences in the file will be converted.